cmd/podman/root.go

```diff
@@ -90,13 +90,6 @@ func init() {
 func Execute() {
 	if err := rootCmd.ExecuteContext(registry.GetContextWithOptions()); err != nil {
 		fmt.Fprintln(os.Stderr, formatError(err))
```

Comment:
when there is an error check `registry.GetExitCode() == 0` and set it to registry.ExecErrorCodeGeneric in this case